app/controllers/person_duplicates/merge_controller.rb

```diff
@@ -9,10 +9,14 @@ module PersonDuplicates
   class MergeController < ApplicationController
     before_action :authorize_action
 
+    helper_method :entry
+
     def new
     end
 
     def create
+      return rerender_form(:unprocessable_entity) unless entry.valid?(context: :merge)
```

Comment:
1. Nice, I have never used this argument in my life (loki-never-saw-this-in-his-life.jpg). Finally a good use-case
2. I suggest to not pass a hash, but only a value...

```suggestion
      return rerender_form(:unprocessable_entity) unless entry.valid?(:merge)
```

I humbly think it reads well enough.